client/src/plugins/update-checks/UpdateChecks.js

```diff
@@ -77,6 +77,8 @@ export default class UpdateChecks extends PureComponent {
 
   async checkLatestVersion(latestUpdateCheckInfo) {
 
+    this.log('Performing a check.');
```

Comment:
Can we use our existing logging facilities? We use `debug` as part of our code base:

```
const log = debug(`UpdateChecks`);

log(`Checking for update ${VERSION}`);

log(`Update available ${VERSION} => ${LATEST_VERSION}`);

log(`No update available for version $⅞VERSIOn`});

Reply:
Cf. https://github.com/camunda/camunda-modeler/search?q=debug&unscoped_q=debug.